README.md

```diff
@@ -1,2 +1,57 @@
-# Sakurabook
-Documentation and guides to Sakurabook Shopify app. 
+# <img src="./img/App%20icon.png" alt="drawing" width="50"/> Welcome to Sakurabook project!
+
+Documentation and guides for Shopify [Sakurabook](https://sakurabook.jp/) app, created by Parkfield.
+
+## Contents
+
+- [Getting Started](./en/getting-started.md)
+- [Setting up the app in Shopify](./en/setting-up-the-app-in-shopify.md)
+- [Usage Guides](./en/usage-guides.md)
+- [Release Notes](./en/release-notes.md)
+- [Links](./en/links.md)
+
+Sakurabook is a plugin for pay-as-you-go reservation service which can be used within the Shopify ecosystem. It can be installed into your Shopify store via [Shopify App Store](https://apps.shopify.com/). It is the first Shopify reservation app that supports the Japanese language. It will fit your business - you can choose if you want to book spaces like rental apartments, studios, or services like therapists, hairdressers, or yoga teachers.
```

Comment:
I read in some guide for Shopify texts, that we should avoid using "the first", "the best", "the only one", etc.

Reply:
IMHO you can use the text used as App listing content
```Shopify’s reservation app supporting Japanese language. Sakurabook will fit your business—you can choose to book spaces like rental apartments, studios, or services like therapies, hairdressers, or yoga classes.```